python/lsst/ip/isr/isrTask.py

```diff
@@ -2068,7 +2080,7 @@ def suspectDetection(self, exposure, amp):
             maskName=self.config.suspectMaskName,
         )
 
-    def maskDefect(self, exposure, defectBaseList):
+    def maskDefect(self, exposure, defectBaseList, numEdgeSuspect=0):
```

Comment:
The docstring on this method is wrong. It was wrong before you started (since it was already setting `SUSPECT` as well as `BAD`), but it's even more wrong now. Could you take this opportunity to fix it?

Reply:
Also consider renaming the method, since it's clearly doing more than masking defects.

Reply:
I've split edge masking into a separate method, and made sure that IsrTask is consistent about calling it.  This also simplifies the main goal of the ticket, as there's a clear way to apply edge masks.